addons/killtracker/CfgEventHandlers.hpp

```diff
@@ -3,3 +3,9 @@ class Extended_PostInit_EventHandlers {
         init = QUOTE(call COMPILE_SCRIPT(XEH_postInit));
     };
 };
+
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_SCRIPT(XEH_preInit));
+    };
+};
```

Comment:
Please add this before the preInit EH, as it's structured that way in every other component.